pkg/kubectl/cmd/util/printing.go

```diff
@@ -126,6 +127,19 @@ func PrinterForCommand(cmd *cobra.Command, outputOpts *printers.OutputOptions, m
 		return nil, err
 	}
 
+	//convert to HumanReadablePrinter will fail after maybeWrapSortingPrinter so we do convert here
+	if !printer.IsGeneric() {
+		// We add handlers to the printer in case it is printers.HumanReadablePrinter.
+		// printers.AddHandlers expects concrete type of printers.HumanReadablePrinter
+		// as its parameter because of this we have to do a type check on printer and
+		// extract out concrete HumanReadablePrinter from it. We are then able to attach
+		// handlers on it.
+		if humanReadablePrinter, ok := printer.(*printers.HumanReadablePrinter); ok {
+			printersinternal.AddHandlers(humanReadablePrinter)
+			printer = humanReadablePrinter
```

Comment:
Any reason to keep this line? as I think printer is already a pointer in this case